Sources/MessagingInApp/MessagingInApp.swift

```diff
@@ -12,6 +12,8 @@ public protocol MessagingInAppInstance: AutoMockable {
     @available(*, deprecated, message: "Parameter organizationId no longer being used. Remove the parameter from your function call to migrate to new function.")
     // sourcery:Name=initializeOrganizationId
     func initialize(organizationId: String)
+
+    func dismissMessage()
```

Comment:
According to the [design proposal for this feature](https://github.com/customerio/issues/issues/9761#issuecomment-1502286550), this function should be a static function added to `MessagingInApp` and not an instance function as it appears here in this PR. 

I was expecting to not see this function here in the protocol. Is there a reason it's needed in the protocol?

Reply:
[This](https://github.com/customerio/customerio-ios/pull/320/files/7e93f6f182adabf5c87ead0d0b20e3178e2952fc#r1200614832) conversation resolves current conversation as well.